sway-parse/src/error.rs

```diff
@@ -74,6 +74,8 @@ pub enum ParseErrorKind {
     UnnecessaryVisibilityQualifier { visibility: Ident },
     #[error("Expected a doc comment.")]
     ExpectedDocComment,
+    #[error("Use `struct`s to define records, instead of `class`.")]
```

Comment:
Hmm, what do you think about 

```suggestion
    #[error("Use the `struct` keyword to define records, instead of `class`.")]
```
?